### ISSUE TITLE
Name and/or url list in "settings for this group"

### ISSUE THREAD

**Issue by yfdyh000** [Quicksaver/Tab-Groups#202]
I see the "settings for this group" are added, since we have automatically catch based on regular, I think this is a simple and useful feature for advanced users.

**Comment by Quicksaver** [comment]
Do you mean that catch rules should also catch tab titles? If so, I agree, that would be nice. I'm leaving that for a future update though, I'd like to see how the catch rules as they are work out in the real world first.

**Comment by yfdyh000** [comment]
Nope, like Copy Link Name & Copy Link URL addons for a group. Mention the catch just goes to show that we don't appear to reject an advanced feature, as well as a possible location.

**Comment by Quicksaver** [comment]
I don't understand then, do you mean like listing all tabs in a way that you can select and copy their links and names in the form of text?

**Comment by yfdyh000** [comment]
Yes.

**Comment by Quicksaver** [comment]
Hmm, with Tab Mix Plus you can already right-click a tab to copy its url, which will work in groups view now as well.

Can you give me an example of a case where you'd need to mass-copy names and urls like that?

**Comment by yfdyh000** [comment]
In fact, I don't urgent need this feature, but I think it was a deserved feature for organization, and it is easy to implement since we have a setting/tools for group.
As a hypothetical demand, which can be used to share all tabs of a group, post the list to blog, forums, etc.
List format might slightly increase complexity, but it can provide a button to bring up a layer to select checkboxes and options, and/or provide a pattern box for advanced users.
